dev/bootstrap.cfg

```diff
@@ -10,6 +10,7 @@ puppetlabs.services.legacy-routes.legacy-routes-service/legacy-routes-service
 puppetlabs.services.puppet-admin.puppet-admin-service/puppet-admin-service
 puppetlabs.trapperkeeper.services.authorization.authorization-service/authorization-service
 puppetlabs.services.versioned-code-service.versioned-code-service/versioned-code-service
+puppetlabs.trapperkeeper.services.scheduler.scheduler-service/scheduler-service
```

Comment:
Ah awesome...I'll fiddle with that and see if I can get an update to get the tests working at least.

I shouldn't have to mess with the tests however for the repl, right? Shouldn't the repl be loading `dev/bootstrap.cfg`? Maybe not...